core/executor/src/lib.rs

```diff
@@ -161,7 +162,7 @@ impl Executor for AxonExecutor {
             let logs_bloom = logs_bloom(r.logs.iter());
             let receipt = tx.encode_receipt(&r, logs_bloom);
 
-            hashes.push(Hasher::digest(&receipt));
+            hashes.push(receipt);
```

Comment:
This variable cannot call `hashes` anymore, it should be `raw_receipts`.

Reply:
it's been encoded, so rename to encode_receipts